include/translated-object.php

```diff
@@ -499,6 +533,25 @@ protected function validate_translations( $translations, $id = 0, $context = 'sa
 		/** @phpstan-var array<non-empty-string, positive-int> $translations */
 		return array_merge( array( $lang->slug => $id ), $translations );
 	}
+
+	/**
+	 * Validates and sanitizes multiple translations arrays.
+	 *
+	 * @since 3.8
+	 *
+	 * @param array[] $translations_arrays An associative array of translations arrays with language code as key and translation ID as value.
```

Comment:
Same as above.

Reply:
Done in 052478dd4cc9fda0e0c909ff163f09151e52ea85